content/docs/command-reference/import-db.md

```diff
@@ -0,0 +1,153 @@
+to install the required packages for the database you want to use.
+
+Some of the recommended packages are shown below, with their expected connection
+strings:
+
+| **Database**        | **PyPI package**                  | **Connection String**                                                                               |
+| ------------------- | --------------------------------- | --------------------------------------------------------------------------------------------------- |
+| **Amazon Redshift** | `sqlalchemy-redshift`             | `redshift+psycopg2://{username}:{password}@{aws_endpoint}:5439/{database_name}`                     |
+| **Big Query**       | `pip install sqlalchemy-bigquery` | `bigquery://{project_id}`                                                                           |
+| **Databricks**      | `databricks-sql-connector`        | `databricks://token:{token}@{hostname}:{port}/{database}?http_path={http_path}`                     |
+| **MySQL**           | `mysqlclient`                     | `mysql://{username}:{password}@{hostname}/{database_name}`                                          |
+| **Oracle**          | `cx_Oracle`                       | `oracle://{username}:{password}@{hostname}/{database_name}`                                         |
+| **PostgreSQL**      | `psycopg2`                        | `postgresql://{username}:{password}@{hostname}/{database_name}`                                     |
+| **Snowflake**       | `snowflake-sqlalchemy`            | `snowflake://{user}:{password}@{account}.{region}/{database}?role={role}&amp;warehouse={warehouse}` |
+| **SQLite**          | -                                 | `sqlite://path/to/file.db`                                                                          |
+| **SQL Server**      | `pyodbc`                          | `mssql+pyodbc://{username}:{password}@{hostname}:{port}/{database_name}`                            |
+| **Trino**           | `trino`                           | `trino://{username}:{password}@{hostname}:{port}/{catalog}`                                         |
+
+DVC uses `sqlalchemy` internally. So DVC should support any SQL databases that
+provide dialects for SQLAlchemy. Refer to their documentation for more details.
+
+## Database Permissions
+
+It is recommended to use an user account with limited access to databases and
+tables and one that only has minimal read-only privileges like running `SELECT`
+queries. Different databases have different approaches to this. Refer to their
+documentation for more details.
+
+## Options
+
+- `-o <path>`, `--out <path>` - specify a `path` to the desired location in the
+  workspace to place the file. The default filename will be `results`, with an
+  extension name added based on `--output-format`.
+
+- `--sql <query>` - SQL query to run. You can also use a connection string set
+  in config, with `--conn`.
```

Comment:
> You can also use a connection string set
  in config, with `--conn`.

I don't follow what this means or why it's mentioned in `--sql`.

Reply:
Leftover from dbt.